test_gaussian_process_complete_config fails

**Describe the bug**
The `test_gaussian_process_complete_config()` test fails due to the presence of an unused keyword argument in the config file.

**To Reproduce**
Steps to reproduce the behavior:
1. cd to the M-LOOP directory
2. Run `pytest -v -k gaussian_process_complete_config`

**Desktop (please complete the following information):**

- OS: Windows 10
- Version: M-LOOP Commit 6ca72cbc1426e9c809245cc5357d863d017138ed

**Additional context**
The issue is that the `gp_training_override_kwargs` argument is passed to `GuassianProcessLearner`, but that option was removed in 845a74b9e0a34b4b17e8b832e4dc9323640ec614. I forgot to update a couple places in the code when doing that. In particular I forgot to remove it from the complete Gaussian process learner config file, which is the reason that this test fails. I also forgot to remove it from the call to `super()` in `GaussianProcessVisualizer.__init__()`, though that doesn't have any effect here. Fortunately these are easy fixes.

Here is the test output of the test for reference:
```python
==================================================================================================== test session starts ===================================================================================================== 
platform win32 -- Python 3.7.6, pytest-5.3.5, py-1.8.1, pluggy-0.13.1 -- C:\Users\user_name\Software\anaconda3\envs\mloop_install_test_2\python.exe
cachedir: .pytest_cache
hypothesis profile 'default' -> database=DirectoryBasedExampleDatabase('C:\\Users\\user_name\\Software\\M-LOOP\\.hypothesis\\examples')
rootdir: C:\Users\user_name\Software\M-LOOP
plugins: hypothesis-5.5.4, arraydiff-0.3, astropy-header-0.1.2, doctestplus-0.5.0, openfiles-0.4.0, remotedata-0.3.2
collected 20 items / 19 deselected / 1 selected

tests/test_examples.py::TestExamples::test_gaussian_process_complete_config FAILED                                                                                                                                      [100%]

========================================================================================================== FAILURES ==========================================================================================================
_____________________________________________________________________________________ TestExamples.test_gaussian_process_complete_config _____________________________________________________________________________________ 

self = <test_examples.TestExamples testMethod=test_gaussian_process_complete_config>

    def test_gaussian_process_complete_config(self):
        controller = mll.launch_from_file(mlu.mloop_path+'/../examples/gaussian_process_complete_config.txt',
                                          interface_type = 'test',
                                          no_delay = False,
>                                         **self.override_dict)

test_examples.py:100:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _  

config_filename = 'c:\\users\\user_name\\software\\m-loop\\mloop/../examples/gaussian_process_complete_config.txt', kwargs = {'console_log_level': 10, 'file_log_level': 30, 'interface_type': 'test', 'no_delay': False, ...}       
file_kwargs = {'gp_training_override_kwargs': False}, interface = <TestInterface(Thread-1, initial)>, controller = <mloop.controllers.GaussianProcessController object at 0x00000238A3112C88>
extras_kwargs = {'visualizations': False}

    def launch_from_file(config_filename,
                         **kwargs):
        '''
        Launch M-LOOP using a configuration file. See configuration file documentation.
    
        Args:
            config_filename (str): Filename of configuration file
            **kwargs : keywords that override the keywords in the file.
    
        Returns:
            controller (Controller): Controller for optimization.
        '''
        try:
            file_kwargs = mlu.get_dict_from_file(config_filename,'txt')
        except (IOError, OSError):
            print('Unable to open M-LOOP configuration file:' + repr(config_filename))
            raise
        file_kwargs.update(kwargs)
        #Main run sequence
        #Create interface and extract unused keywords
        interface = mli.create_interface(**file_kwargs)
        file_kwargs = interface.remaining_kwargs
        #Create controller and extract unused keywords
        controller = mlc.create_controller(interface, **file_kwargs)
        file_kwargs = controller.remaining_kwargs
        #Extract keywords for post processing extras, and raise an error if any keywords were unused.
        extras_kwargs = _pop_extras_kwargs(file_kwargs)
        if file_kwargs:
            logging.getLogger(__name__).error('Unused extra options provided:' + repr(file_kwargs))
>           raise ValueError
E           ValueError

..\mloop\launchers.py:42: ValueError
---------------------------------------------------------------------------------------------------- Captured stdout call ---------------------------------------------------------------------------------------------------- 
INFO     M-LOOP version 3.2.1
DEBUG    M-LOOP Logger configured.
DEBUG    Creating interface.
DEBUG    Setting default landscapes
INFO     Using the test interface with the experiment.
DEBUG    Controller init completed.
DEBUG    Learner init completed.
DEBUG    Random learner init completed.
DEBUG    Learner init completed.
ERROR    Unused extra options provided:{'gp_training_override_kwargs': False}
----------------------------------------------------------------------------------------------------- Captured log call ------------------------------------------------------------------------------------------------------ 
INFO     mloop:utilities.py:86 M-LOOP version 3.2.1
DEBUG    mloop:utilities.py:87 M-LOOP Logger configured.
DEBUG    mloop.interfaces:interfaces.py:82 Creating interface.
DEBUG    mloop.testing:testing.py:33 Setting default landscapes
INFO     mloop.interfaces:interfaces.py:40 Using the test interface with the experiment.
DEBUG    mloop.controllers:controllers.py:235 Controller init completed.
DEBUG    mloop.learners.1:learners.py:185 Learner init completed.
DEBUG    mloop.learners.1:learners.py:469 Random learner init completed.
DEBUG    mloop.learners.2:learners.py:185 Learner init completed.
ERROR    mloop.launchers:launchers.py:41 Unused extra options provided:{'gp_training_override_kwargs': False}
============================================================================================== 1 failed, 19 deselected in 3.37s ============================================================================================== 

```

--- a/examples/gaussian_process_complete_config.txt
+++ b/examples/gaussian_process_complete_config.txt
@@ -31,6 +31,5 @@ training_type = 'random'               #training type can be random, differentia
 first_params = [1.9, -1.0]             #first parameters to try in initial training
 gp_training_filename = None            #filename for training from previous experiment
 gp_training_file_type = 'pkl'          #training data file type
-gp_training_override_kwargs = False    #whether to override user-supplied keyword options with values set in training archive.
 
 #if you use nelder_mead for the initial training source see the CompleteNelderMeadConfig.txt for options. 

--- a/mloop/visualizations.py
+++ b/mloop/visualizations.py
@@ -815,7 +815,6 @@ class GaussianProcessVisualizer(mll.GaussianProcessLearner):
 
         super(GaussianProcessVisualizer, self).__init__(gp_training_filename = filename,
                                                         gp_training_file_type = file_type,
-                                                        gp_training_override_kwargs=True,
                                                         update_hyperparameters = False,
                                                         learner_archive_filename=None,
                                                         **kwargs)

